<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox1">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>30</y>
      <width>211</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>1. Calibration</string>
    </property>
    <widget class="QPushButton" name="Find_Corners">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>1.1 Find Corners</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Find_Intrinsic">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>1.2 Find Intrinsic</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Find_Distortion">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>1.4 Find Distortion</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Show_Result">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>1.5 Show Result</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>171</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>1.3 Find Extrinsic</string>
     </property>
     <widget class="QPushButton" name="Find_Extrinsic">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>65</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>1.3 Find Extrinsic</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Calibration_InputBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>31</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Image:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>211</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>2. Augmented Really</string>
    </property>
    <widget class="QPushButton" name="Show_Words_on_Board">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>2.1 Show Words on Board</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Show_Words_Vertically">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>2.2 Show Words Vertically</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="AR_InputBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>61</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>211</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>3. Stereo Disparity Map</string>
    </property>
    <widget class="QPushButton" name="Stereo_Disparity_Map">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>3.1 Stereo Disparity Map</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>30</y>
      <width>211</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>4. SIFT</string>
    </property>
    <widget class="QPushButton" name="Keypoints">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>71</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>4.1 Keypoints</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Matched_keypoints">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>151</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>4.2 Matched keypoints</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Warp_Image">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>4.3 Warp Image</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>932</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
